setupAndVerifyJDK.md

```diff
@@ -105,3 +105,37 @@ As output you should get something like this:
 ```
     javac 9-ea
 ```
+
```

Comment:
http://osxdaily.com/2015/08/06/make-symbolic-links-command-line-mac-os-x/ they seem to have the same syntax, lets hope it works on the MacOSX as well.